src/_includes/css/header.css

```diff
@@ -7,7 +7,7 @@
     margin: 0;
     text-wrap: balance;
 
-    + p {
+    & + p {
```

Comment:
Is there a way to disable this behavior? `&` character is not needed any more for nested rules in CSS.

Reply:
Currently no. If we delete `&`, it will produce syntax errors.

Reply:
I think it would be more intuitive to use the `&` symbol in nesting css.

Reply:
It's added only if the nesting selector starts with characters like `+` or `>`. But not if the nesting selector is, for example, `footer { a { ... }}`.